packages/e2e-tests/puppeteer/__tests__/web3torrent/seed-download.test.ts

```diff
@@ -37,7 +37,7 @@ describe('Web3-Torrent Integration Tests', () => {
     await loadDapp(web3tTabA, 0, true);
     await loadDapp(web3tTabB, 0, true);
 
-    await web3tTabA.goto('http://localhost:3000/file/new', {waitUntil: 'load'});
```

Comment:
Is it possible to refer to constants or url generators defined in web3torrent to construct these?

Reply:
this URL is no longer up to date with the URL for the upload page in web3torrent. Please, use the ones on [here](https://github.com/statechannels/monorepo/blob/master/packages/web3torrent/src/routes.ts)